extensions/jni/JVMCreator.h

```diff
@@ -40,7 +40,7 @@ namespace jni {
 class JVMCreator : public minifi::core::CoreComponent {
  public:
 
-  explicit JVMCreator(const std::string &name, utils::Identifier uuid = utils::Identifier())
+  explicit JVMCreator(const std::string &name, const utils::Identifier& uuid = {})
```

Comment:
inconsistent `&` placing

Reply:
Fixed in 986bca3bf8ae4897cf73ce1fbe6530015f79989e